src/lib/components/ReactSearchKit/ReactSearchKit.js

```diff
@@ -26,6 +26,7 @@ export class ReactSearchKit extends Component {
         ? props.urlHandlerApi.customHandler ||
           new UrlHandlerApi(props.urlHandlerApi.overrideConfig)
         : null,
+      updateQueryStateFromResponse: props.updateQueryStateFromResponse,
```

Comment:
IMHO, this should be handled by the response serializer only, so an option there and not here at the RSK app. WDYT?

Reply:
Hmm but the serializer doesn't know how to update the state...I need to modify the way the `executeQuery` action is working...what did you have in mind?